packages/x-charts/src/ChartsTooltip/ChartsTooltip.tsx

```diff
@@ -21,6 +21,13 @@ import { ChartsItemContentProps, ChartsItemTooltipContent } from './ChartsItemTo
 import { ChartsAxisContentProps, ChartsAxisTooltipContent } from './ChartsAxisTooltipContent';
 import { ChartsTooltipClasses, getChartsTooltipUtilityClass } from './chartsTooltipClasses';
 
+export type PopperProps = BasePopperProps & {
```

Comment:
Nitpick: IMHO, this type could have also been prefixed with `Charts` to avoid the conflict with `base` and `material` types.

Reply:
That could be better indeed, is there a pattern we follow in such cases? Like, both material and base are `PopperProps`, so there is already some confusion there 🤔

Reply:
There are no hard rules and material/base are not an examples of making types unique.
For Charts we somewhat settled on trying to prefix types (especially generic ones) with `Charts` to avoid further confusion.
For example, we have the `ChartsGridProps` instead of `GridProps` to avoid potential errors. 🙈 
Maybe @alexfauquette has a more clear vision/direction for type naming? 🤔

Reply:
Since the component name is `ChartsTooltipRoot` I assume we could have named it `ChartsTooltipRootProps` But those naming are not ideal. They would benefit an entire renaming to harmonize the notion of tooltip/popper

I do not expect user to use it, or if they do the confusion with the core type should not be a big deal since they are quite similar